tests/table-integration.js

```diff
@@ -78,6 +79,18 @@ describe( 'Table feature – integration', () => {
 				[ '<blockQuote><paragraph>bar[]</paragraph></blockQuote>' ]
 			] ) );
 		} );
+
+		it.only( 'merges elements without throwing errors', () => {
```

Comment:
`.only()`

Reply:
We should have a test whether some test conains `/(describe|it)\.only\(/` on the CI 😄

Reply:
Coverage drops below the "critical" level. It should be a warning for us. Unfortunately, @Reinmar was faster then I read the email :D